zsh_completion/_kerl

```diff
@@ -4,7 +4,7 @@
 # Description
 # -----------
 #
-#  Completion script for kerl (http://github.com/spawngrid/kerl
+#  Completion script for kerl (https://github.com/spawngrid/kerl)
```

Comment:
Has no real "influence" apart from showing up in searches.

Reply:
might as well make it `kerl/kerl` then

Reply:
Sure. I didn't want to change it, on purpose, for history reasons.

Reply:
7f5f892